mobile/lib/modules/backup/services/backup.service.dart

```diff
@@ -236,7 +267,13 @@ class BackupService {
           var response = await req.send(cancellationToken: cancelToken);
 
           if (response.statusCode == 201) {
-            singleAssetDoneCb(entity.id, deviceId);
+            var responseBody = await response.stream.bytesToString();
+            var uploadResponse =
+                AssetFileUploadResponseDto.fromJson(jsonDecode(responseBody));
```

Comment:
instead of sending a body from the server.. and needing to parse it here, could we simply use different HTTP status codes to signal (upload ok vs. upload ok, but duplicated)?
should be simpler and a little bit faster